source/manual/notify-emergency-alerts.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#govuk-notify-emergency-alerts"
+title: What is GOV.UK Notify Emergency Alerts?
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+section: Notify Emergency Alerts
+important: true
```

Comment:
What does `important: true` do? I've scanned govuk-developer-docs, tech-docs-gem and tech-docs-template and couldn't see any reference to functionality.

Reply:
It controls what gets shown in this [`Common tasks`](https://docs.publishing.service.gov.uk/manual.html#common-tasks) section I believe:

https://github.com/alphagov/govuk-developer-docs/blob/ec5e0afaa3df4daefc2d1cebc8a34b4872d83486/app/manual.rb#L10-L16

Reply:
Ah yes - https://github.com/alphagov/govuk-developer-docs/blob/ec5e0afaa3df4daefc2d1cebc8a34b4872d83486/app/manual.rb#L45